README.md

```diff
@@ -125,43 +127,40 @@ Then copy the `id` property value. It will be used in the next step as `SUBSCRIP
 
 ### 4. Create Variable Groups
 
-Get you service principal, your ACR credentials, and fill the following empty variables. Then, run this block on Bash:
+Ensure below steps are complete before creating variable groups on Azure Devops:
+- Azure Service Connectione name - Use the service principle from above to set up service connection
+- Azure Key Vault name and set up below secrets within the vault with keys specified below:
+  - Set up the ARM client secret with key 'arm_client_secret' assigned

```

Comment:
Is `arm_client_secret` correct? I tried to setup and it I'm getting errors:
![image](https://user-images.githubusercontent.com/13934447/89327452-55a16100-d662-11ea-92f5-275db8122459.png)

Reply:
ah yes..thats a typo. I have just double checked my key vault and the key is indeed set up as **arm-client-secret** . Will correct it thanks.

Reply:
Cool. Let me know if you need any help with the PR :)

Reply:
thanks 🙂 . Its been corrected now.